src/screens/SendConfirmationSheet.tsx

```diff
@@ -242,28 +261,132 @@ export default function SendConfirmationSheet() {
     return get(contacts, `${[toLower(toAddress)]}`);
   }, [contacts, toAddress]);
 
-  const [checkboxes, setCheckboxes] = useState([
-    {
-      checked: false,
-      label: lang.t(
-        'wallet.transaction.checkboxes.im_not_sending_to_an_exchange'
-      ),
-    },
-    {
-      checked: false,
-      label: lang.t(
-        'wallet.transaction.checkboxes.has_a_wallet_that_supports',
-        {
-          networkName: capitalize(network),
+  const uniqueTokenType = getUniqueTokenType(asset);
+  const isENS = uniqueTokenType === 'ENS';
+
+  const [checkboxes, setCheckboxes] = useState<Checkbox[]>(
+    isENS
+      ? ([
+          !hasClearProfileInfo(ensProfile) && {
+            checked: false,
+            id: 'clear-records',
+            label: 'Clear profile information',
+          },
+          !doesNamePointToRecipient(ensProfile, toAddress) && {
+            checked: false,
+            id: 'set-address',
+            label: 'Point this name to the recipient’s wallet address',
```

Comment:
i think we're missing the case were !!doesNamePointToRecipient

https://www.figma.com/file/O2iH8h2H00ycicwUHIfBah/profiles-spec?node-id=586%3A9591

Reply:
Is this the valid design? I thought we take the right-most ones.